torch/fx/experimental/proxy_tensor.py

```diff
@@ -235,6 +235,11 @@ def fetch_tensor_proxy(tracer):
 HANDLED_TYPES = (torch.Tensor, torch.nn.Parameter)
 
 def proxy_call(proxy_mode, func, args, kwargs):
+    # `__torch_dispatch__` is only called on torch ops, which must have `overloadpacket` attr
+    # hence, the lack of such an attribute indicates an `external_call`, perhaps a function
+    # decorated with `@torch.tx.wrap`
+    external_call = not hasattr(func, "overloadpacket")
```

Comment:
This seems like a hokey way to test for this. Why don't we do an isinstance test, or even just have an optional kwarg for indicating external calls?

Reply:
I initially considered external_call kwarg, but noted that a user may try to wrap a torch op for whatever reason.

I'll go with `isinstance(.., OpOverload)`?